lib/bap/bap.mli

```diff
@@ -5512,6 +5512,7 @@ module Std : sig
 
     (** [entry_point addr] is an address from which a kernel should start *)
     val entry_point : t -> addr
+    
```

Comment:
```suggestion

```

Reply:
You have inserted accidentally four whitespaces on the empty line, which makes the style checker [unhappy][1] :)

[1]: https://github.com/BinaryAnalysisPlatform/bap/pull/1283/checks#step:4:52